mmcls/models/backbones/mobileone.py

```diff
@@ -0,0 +1,514 @@
+            weight_scale = F.pad(weight_scale, [pad, pad, pad, pad])
+
+        weight_norm, bias_norm = 0, 0
+        if self.branch_norm:
+            tmp_conv_bn = self._norm_to_conv(self.branch_norm)
+            weight_norm, bias_norm = self._fuse_conv_bn(tmp_conv_bn)
+
+        return (weight_conv + weight_scale + weight_norm,
+                bias_conv + bias_scale + bias_norm)
+
+    def _fuse_conv_bn(self, branch):
+        """Fuse the parameters in a branch with a conv and bn.
+
+        Args:
+            branch (mmcv.runner.Sequential): A branch with conv and bn.
+
+        Returns:
+            tuple[torch.Tensor, torch.Tensor]: The parameters obtained after
+                fusing the parameters of conv and bn in one branch.
+                The first element is the weight and the second is the bias.
+        """
+        if branch is None:
+            return 0, 0
+        kernel = branch.conv.weight
+        running_mean = branch.norm.running_mean
+        running_var = branch.norm.running_var
+        gamma = branch.norm.weight
+        beta = branch.norm.bias
+        eps = branch.norm.eps
+
+        std = (running_var + eps).sqrt()
+        fused_weight = (gamma / std).reshape(-1, 1, 1, 1) * kernel
+        fused_bias = beta - running_mean * gamma / std
+
+        return fused_weight, fused_bias
+
+    def _norm_to_conv(self, branch_nrom):
+        """Convert a norm layer to a conv-bn sequence towards
+        ``self.kernel_size``.
+
+        Args:
+            branch (nn.BatchNorm2d): A branch only with bn in the block.
+
+        Returns:
+            (mmcv.runner.Sequential): a sequential with conv and bn.
+        """
+        input_dim = self.in_channels // self.groups
+        conv_weight = torch.zeros(
+            (self.in_channels, input_dim, self.kernel_size, self.kernel_size),
+            dtype=branch_nrom.weight.dtype)
+
+        for i in range(self.in_channels):
+            conv_weight[i, i % input_dim, self.kernel_size // 2,
+                        self.kernel_size // 2] = 1
+        conv_weight = conv_weight.to(branch_nrom.weight.device)
+
+        tmp_conv = self.create_conv_bn(kernel_size=self.kernel_size)
+        tmp_conv.conv.weight.data = conv_weight
+        tmp_conv.norm = branch_nrom
+        return tmp_conv
+
+
+@BACKBONES.register_module()
+class MobileOne(BaseBackbone):
+    """MobileOne backbone.
+
+    A PyTorch impl of : `An Improved One millisecond Mobile Backbone
+    <https://arxiv.org/pdf/2206.04040.pdf>`_
+
+    Args:
+        arch (str | dict): MobileOne architecture. If use string, choose
+            from 's0', 's1', 's2', 's3' and 's4'. If use dict, it should
+            have below keys:
+
+            - num_blocks (Sequence[int]): Number of blocks in each stage.
+            - width_factor (Sequence[float]): Width deflator in each stage.
+            - num_conv_branches (Sequence[float]): Number of conv branches
+                 in each stage.
```

Comment:
The item in a list should be aligned left.
```suggestion
            - num_conv_branches (Sequence[float]): Number of conv branches
              in each stage.
```